float64 frontLeft
float64 frontRight
float64 backLeft
float64 backRight

